atc/db/team.go

```diff
@@ -955,33 +955,37 @@ func (t *team) FindCheckContainers(logger lager.Logger, pipelineRef atc.Pipeline
 		return nil, nil, nil
 	}
 
-	pipelineResourceTypes, err := pipeline.ResourceTypes()
-	if err != nil {
-		return nil, nil, err
-	}
+	// find any builds that have resource_id = resource.id
+	// Find any containers that have build_id ^
+
+	// XXX: THIS DOES NOT WORK!
```

Comment:
Leaving this as a reminder since it's not listed in the TODOs.

p.s. is there a reason we need to `FindOrCreateResourceConfig` here? Could we instead use the condition:
```
sq.Eq{"rccs.resource_config_id": resource.ResourceConfigID()}
```

I guess one implication of that is that we'd need to have called `PointToCheckedConfig` at least once in the check delegate (i.e. at least one check had to have completed, either successfully or unsuccessfully).